screens/AccountClosureForm.js

```diff
@@ -98,67 +81,72 @@ class AccountClosureForm extends React.Component {
       <DismissKeyboard>
       	<ScrollView scrollIndicatorInsets={{ right: 1 }}>
 
-          {/* Cash */}
-          <View style={styles.sectionBar}>
-            <View>
-              <Text style={styles.sectionBarText}>
-                {t('cashSection')}
-              </Text>
-            </View>
-          </View>
-
-          <View style={[styles.tableRowContainerWithBorder]}>
-            <View style={[styles.tableCellView, {flex: 2}]}>
-              <Text style={[styles.fieldTitle]}>
-                {t('startingCash')}
-              </Text>
-            </View>
-            <View style={[styles.tableCellView, {flex: 3, justifyContent: 'flex-end'}]}>
-              <Text>${mostrecentShift.open.balance}</Text>
-            </View>
-          </View>
-
-          <View style={[styles.tableRowContainerWithBorder]}>
-            <View style={[styles.tableCellView, {flex: 2}]}>
-              <Text style={[styles.fieldTitle]}>
-                {t('totalCashTransitionAmt')}
-              </Text>
-            </View>
-            <View style={[styles.tableCellView, {flex: 3, justifyContent: 'flex-end'}]}>
-              <Text>${closingShiftReport.totalByPaymentMethod.hasOwnProperty('CASH') ? closingShiftReport.totalByPaymentMethod.CASH.orderTotal : 0}
-              </Text>
-            </View>
-          </View>
-
-          <View style={styles.tableRowContainerWithBorder}>
-            <View style={[styles.tableCellView, {flex: 2}]}>
-              <Text style={[styles.fieldTitle]}>
-                {t('totalCashInRegister')}
-              </Text>
-            </View>
-            <View style={[styles.tableCellView, {flex: 3, justifyContent: 'flex-end'}]}>
-              <Field
-                name="cashclosingBalance"
-                component={InputText}
-                keyboardType={`numeric`}
-                validate={isRequired}
-                placeholder={t('enterAmount')}
-                defaultValue={String(enteredCashAmount)}
-              />
-            </View>
-          </View>
-
-          <View style={styles.tableRowContainerWithBorder}>
-            <View style={{flex: 1}}>
-              <Field
-                name="cashunbalanceReason"
-                component={InputText}
-                placeholder={t('remark')}
-                height={35}
-              />
-            </View>
-          </View>
-
+			{/* Cash */}
```

Comment:
The latest origin/master has changed this section and it seems like you are overwriting to previous implementation. Please revert the changes here.